### DISCUSSION TITLE
Logging generated Sql

### DISCUSSION THREAD

**Discussion by 97lynk** [toluaina/pgsync#281] [Q&A] [answered]
Hi @toluaina 
You mentioned to "Behind the scenes, PGSync is generating advanced queries for you such as." on README, how to logging that sql query?
Thank you very much.

**Comment by toluaina** [comment, marked as answer]
Hello. @97lynk 

You can run pgsync with verbose to show the queries
add `-v`

**Reply by 97lynk** [reply]
Thank you for the answer.
